io.github.jeffshee.Hidamari.json

```diff
@@ -0,0 +1,240 @@
+{
+    "app-id": "io.github.jeffshee.Hidamari",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "42",
+    "sdk": "org.gnome.Sdk",
+    "command": "hidamari",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--device=dri",
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--filesystem=home:ro",
+        "--filesystem=xdg-config/hidamari",
```

Comment:
Yous shoudl respect the [XDG Base Directry Spec](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html). Flatpak sets some env vars to point to custom directories, so you should not save in .config, if the App is run as Flatpak.

Reply:
Hi, I'm not sure about this. Aren't apps writing their config data in `~/.config/foobar` directory? What is the alternative you would suggest?

Reply:
if you write to `~/.config/foobar` you are doing it wrong. If you write to what XDG gives you (which CAN be that), then you don't need this.

I do suggest your remove it.

Reply:
Solved. Now it is using `$XDG_CONFIG_HOME/hidamari`